sender/snmp.go

```diff
@@ -48,64 +51,65 @@ func (x *SNMP) init() {
 	}
 
 	x.g.Target = x.Target
-	x.g.Connect()
+	x.err = x.g.Connect()
 }
 
 func (x *SNMP) Send(c *skogul.Container) error {
 	x.r.Do(func() {
 		x.init()
 	})
 
-	if x.err != nil {
-		return x.err
+	var pdutypes []gosnmp.SnmpPDU
+	if len(x.PduTypes) > 0 {
+		pdutypes = x.PduTypes
+	} else {
+		pdutypes = []gosnmp.SnmpPDU{
+			{
+				Value: ".1.3.6.1.4.1.12748.2023.0.888",
+				Name:  ".1.3.6.1.6.3.1.1.4.1.0",
+				Type:  gosnmp.ObjectIdentifier,
+			},
```

Comment:
This is still hard coded.

I did a little research. Turns out  .1.3.6.1.6.3.1.1.4.1 is literally "snmpTrapOID" oid, and always part of a trap, however, the value needs to not be hard-coded, even if it can be overridden now.  The value you provide is Telenor-specific, and the .2023 makes it very implementation-specific.

I suggest changing the configurable PduTypes to say "SnmpTrapOID" and being of type string, and then using that as value.

E.g.:

```        
	pdutypes = []gosnmp.SnmpPDU{
			{
				Value: x.SnmpTrapOID,
				Name:  ".1.3.6.1.6.3.1.1.4.1.0",
				Type:  gosnmp.ObjectIdentifier,
			},
```

That does mean this wont work without configuration, but it shouldn't. The value here should always be configured, it's very much site-local and not something where we can assume a sensible global default exists. 

See:

https://oidref.com/1.3.6.1.6.3.1.1.4.1
https://oidref.com/1.3.6.1.4.1.12748

A bonus would be to add to Verify() that it makes sure that the value is: 
1. Actually provided.
2. Matches [0-9.]+